Makefile

```diff
@@ -6,25 +6,33 @@ gg:
 	MEMORY_FULL=32000 \
 	EVAL_DEPTH=7 \
 	TAX_TYPE=greengenes \
+	TIME_EXTRACT_READS=45 \
+	TIME_TRAIN_515_806=120 \
+	TIME_TRAIN_FULL=230 \
+	TIME_TRAIN_WEIGHTS=75 \
 		./train.sh
 
 silva:
-	CLASSIFIER_NAME=silva-138-99 \
+	CLASSIFIER_NAME=silva-138-1-99 \
 	MEMORY_515_806=64000 \
 	MEMORY_FULL=64000 \
 	EVAL_DEPTH=7 \
 	TAX_TYPE=silva \
+	TIME_EXTRACT_READS=75 \
+	TIME_TRAIN_515_806=3300 \
+	TIME_TRAIN_FULL=8640 \
+	TIME_TRAIN_WEIGHTS=1440 \
 		./train.sh
 
 getgg:
 	CLASSIFIER_NAME=gg-13-8-99 \
 		./get_gg.sh
 
 getsilva:
-	VERSION=138 \
+	VERSION=138.1 \
```

Comment:
drive-by comment: would it be possible to skip explicit version specification? and use the latest version supported by RESCRIPt instead? The `VERSION` is also used in output filenames, but maybe this could be scraped from provenance (or just import the default version specification from rescript)